index.test.js

```diff
@@ -85,6 +85,37 @@ describe('Login to ECR', () => {
             expect.anything());
     });
 
+    test('outputs the registry ID if a single registry is provided in the input', async () => {
+        core.getInput = jest.fn().mockReturnValueOnce('111111111111');
```

Comment:
should the mocked return value be `registries: { with: [1111111111] }` ?

Reply:
Do you mean to mock the configuration: 
```
      - name: Login to Amazon ECR
        id: login-ecr
        uses: aws-actions/amazon-ecr-login@v1
        with:
          registries: 1111111111
```

The GH actions toolkit takes care of parsing the config, and then the action just does core.getInput('registries')